src/App.js

```diff
@@ -26,11 +27,23 @@ function App() {
   const _height = 500;
   const _width = 500;
 
+  const [gameOver, setGameOver] = useState(false);
   const [pose, setPose] = useState(null);
   const [maxScore, setScore] = useState(0);
   const [glasses, setGlasses] = useState(false);
   const [vectors, setVectors] = useState(false);
 
+  const [leaders, setLeaders] = useState([
+    { name: 'Zack', score: 23 },
+    { name: 'Kelly', score: 18 },
+    { name: 'Screech', score: 1 },
+    { name: 'Slater', score: 21 },
+    { name: 'Jessie', score: 22 },
+    { name: 'Lisa', score: 13 },
+    { name: 'Mr. B', score: 9 },
+    { name: 'Tori', score: 13 },
+  ]);
+
   const handleKeyPress = {
     GLASSES_MODE: () => setGlasses(true),
```

Comment:
what about 
`setGlasses(!glasses)` for true toggleability(tm)

Reply:
You can't toggle set state because it will always be false. But we can do something like this:

```
glasses ? setState(false) : setState(true);
```

I will update both and remove reset.

Reply:
...`setVectors` isn't working for me in the hotkey handlers. Can we make a new issue for this?